internal/cli/atlas_clusters_update.go

```diff
@@ -69,8 +67,8 @@ func (opts *atlasClustersUpdateOpts) cluster() (*atlas.Cluster, error) {
 	if opts.filename != "" {
 		cluster = new(atlas.Cluster)
 		err = file.Load(opts.fs, opts.filename, cluster)
-		if opts.name != "" {
-			cluster.Name = opts.name
+		if opts.name == "" {
```

Comment:
flipping this so we always use the command property and not the cluster value due to the change on the store signature